scripts/templates/api.tpl.ejs

```diff
@@ -28,7 +28,9 @@ title: <?= method.command ?>
 | Name | Type | Details |
 | ---- | ---- | ------- |
 <? method.paramTags.forEach((paramTag) => {
-    var paramKey = (paramTag.types && paramTag.typesDescription) ? paramTag.typesDescription : paramTag.type
+    var paramKey = paramTag.types
+        ? paramTag.types.map((type) => `<code>${type.replace(/>/g, "&gt;").replace(/</g, "&lt;")}</code>`).join('|')
```

Comment:
Is it possible to get value without html tags from `type`?
Sorry to say I don't know what is `type` Object.

Reply:
type could be `Promise.<any>` and I am replacing the `<` and `>` here to not quit the `</code>` to early.